run/events-pubsub/app.js

```diff
@@ -23,6 +23,7 @@ app.post('/', (req, res) => {
     ? Buffer.from(pubSubMessage.data, 'base64').toString().trim()
     : 'World';
   
+  console.log(`Hello, ${name}! ID: ${req.get('ce-id') || ''}`);
   res.send(`Hello, ${name}! ID: ${req.get('ce-id') || ''}`);
```

Comment:
Should we DRY up the response values? Should there be a comment noting the message is sent in the response for local testing?

```suggestion
  const msg = `Hello, ${name}! ID: ${req.get('ce-id') || ''}`;
  console.log(msg);
  res.send(msg);
```

Reply:
+1